docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch.md

```diff
@@ -452,14 +443,17 @@ _sourceCategory=ecs* (DeleteCluster or DeleteService or DeregisterContainerInsta
 
 ### Install the Sumo Logic App 
 
-Now that you have set up a [collection](/docs/integrations/amazon-aws/elastic-container-service) for Amazon ECS, install the Sumo Logic App for Amazon ECS to use the pre-configured searches and dashboards that provide visibility into your environment for real-time analysis of overall usage.
+Now that you have set up a collection for Amazon ECS with Container Insights and CloudWatch, install the Sumo Logic app for Amazon ECS with Container Insights and CloudWatch to use the pre-configured searches and dashboards that provide visibility into your environment for real-time analysis of overall usage.
 
 To install the app:
 
 Locate and install the app you need from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
 
 1. From the **App Catalog**, search for and select the app. 
-2. Select the **With Container Insights and Traces** version and click **Add to Library**. Version selection applies only to a few apps currently. For more information, see the [Install the Apps from the Library](/docs/get-started/library).
+2. Select the **With Container Insights and Traces** version and click **Add to Library**. 
+:::note
+Version selection applies only to a few apps currently. For more information, see the [Install the Apps from the Library](/docs/get-started/library).
+:::
```

Comment:
```suggestion
```

Reply:
Dev guys needed note note. So as per the request from them I have changed the step from a note.

Reply:
@JV0812 From what I understand, this note is obsolete. It was written 1-2 years ago Docs-wide, when App versioning was just coming out in Beta. But now feature is ubiquitous.

Reply:
Okay, got it. Thanks for this info @kimsauce